cypress/support/commands.js

```diff
@@ -0,0 +1,27 @@
+// ***********************************************
```

Comment:
looks like this whole file can go unless you're going to need it for something else soon.